src/components/BionicReaderPage/index.tsx

```diff
@@ -43,29 +43,29 @@ const useTextProcessing = () => {
           <span className="bio-letter">{preElem.slice(0, mid)}</span>
           {preElem.slice(mid)}
         </>
-      );
-    });
+      )
+    })
 
-    return listText;
-  };
+    return listText
+  }
 
   const onClickButton = async (e: React.MouseEvent<HTMLButtonElement>) => {
-    setPretext("processing...");
-    setListPrepText([]);
+    setPretext('processing...')
+    setListPrepText([])
 
-    console.log("wait...");
-    setIsDisabled(true);
+    console.log('wait...')
+    setIsDisabled(true)
 
-    const listPrepText = processData();
-    setPretext("processing...");
+    const listPrepText = processData()
+    setPretext('processing...')
 
-    await timeout(2000);
+    await timeout(2000)
```

Comment:
you should remove this. why would you wait for something, that does not need any time?

Reply:
end-users like animations